src/Service.Tests/Configuration/ConfigurationTests.cs

```diff
@@ -2717,6 +2719,91 @@ public async Task OpenApi_EntityLevelRestEndpoint()
             Assert.IsFalse(componentSchemasElement.TryGetProperty("Publisher", out _));
         }
 
+        /// <summary>
+        /// This test validates that DAB properly creates and returns a nextLink with a single $after
+        /// query parameter when sending paging requests.
+        /// The first request initiates a paging workload, meaning the response is expected to have a nextLink.
+        /// The validation occurs after the second request which uses the previously acquired nextLink
+        /// This test ensures that the second request's response body contains the expected nextLink which:
+        /// - is base64 encoded and NOT URI escaped e.g. the trailing "==" are not URI escaped to "%3D%3D"
+        /// - is not the same as the first response's nextLink -> DAB is properly injecting a new $after query param
+        /// and updating the new nextLink
+        /// - does not contain a comma (,) indicating that the URI namevaluecollection tracking the query parameters
+        /// did not come across two $after query parameters. This addresses a customer raised issue where two $after
+        /// query parameters were returned by DAB.
+        /// </summary>
+        [TestMethod]
+        [TestCategory(TestCategory.MSSQL)]
+        public async Task ValidateNextLinkUsage()
+        {
+            // Arrange - Setup test server with entity that has >100 records enabling paging.
```

Comment:
I could reproduce this even with only 3 records in my table, by using the $first clause value=1, so doesnt have to be >100 records.
The second response contained a `nextLink` with 2 after params

Reply:
yeah i realize i was using $first clause too and can have the same behavior. I'll revise the comment so it doesn't confuse anyone.